web/src/features/fireWeather/components/AccuracyVariablePicker.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { FormControl, InputLabel, Select, MenuItem } from '@material-ui/core'
+
+export enum AccuracyWeatherVariableEnum {
+  'Relative Humidity' = 1,
+  'Temperature'
+}
+
+const useStyles = makeStyles({
+  variableDropdown: {
+    minWidth: 250,
+    margin: '8px',
+    marginLeft: '20px'
+  },
+  select: {
+    color: 'white'
+  },
+  '& .MuiFormLabel-root.Mui-focused': {
+    color: 'white'
+  }
+})
+
+interface Props {
+  selectedWxVariable: AccuracyWeatherVariableEnum
+  changeHandler: any
```

Comment:
Lint is complaining about the any, I think this can be a function that accepts an event and returns void.